src/WebAppDIRAC/WebApp/handler/ResourceSummaryHandler.py

```diff
@@ -312,6 +311,17 @@ def web_action(self, action=None, **kwargs):
 
     def web_getResourceSummaryData(self, name=None, status=None, elementType=None, statusType=None, tokenOwner=None):
         """This method returns the data required to fill the grid."""
+        if name:
+            name = list(json.loads(name))
```

Comment:
What is the `list()` doing here? I struggle to see how could be different to just:
```suggestion
            name = json.loads(name)
```
(same for the later lines)